README.md

```diff
@@ -367,11 +367,18 @@ To use this library with [Azure OpenAI](https://learn.microsoft.com/en-us/azure/
 class instead of the `OpenAI` class.
 
 > [!IMPORTANT]
-> The Azure API shape differs from the core API shape which means that the static types for responses / params
+> The Azure API shape slightly differs from the core API shape which means that the static types for responses / params
 > won't always be correct.
 
 ```ts
-const openai = new AzureOpenAI();
+import { AzureOpenAI } from 'openai';
```

Comment:
I'm realising now that we don't mention the environment variables the azure client supports, e.g. `AZURE_OPENAI_ENDPOINT` & `AZURE_OPENAI_API_KEY`, we should mention these in the README as well. (doesn't block this PR)